packages/analytics/dashboard-renderer/src/components/TimeseriesChartRenderer.vue

```diff
@@ -9,6 +9,7 @@
         :chart-data="data"
         :chart-options="options"
         legend-position="bottom"
+        tooltip-title=""
```

Comment:
@filipgutica same here

Reply:
Bring it back :) Sorry for the troubles.